docs/docs/customization/buttons.md

```diff
@@ -0,0 +1,52 @@
+Pathfora modules can have up to two action buttons "confirm" and "cancel". You can set custom text for these buttons as well as select to hide one or both of them. See the [callbacks](callbacks.md) section for how to add javascript callbacks on these buttons.
```

Comment:
Action Buttons may be a better name for this page. That way it's less likely to get confused with the Button layout.